Packs/CortexXDR/Integrations/CortexXDRIR/README.md

```diff
@@ -157,6 +157,7 @@ To setup the mirroring follow these instructions:
   - The default playbook of the *Cortex XDR Incident* incident type is not *Cortex XDR Incident Sync*, change it to a 
      different playbook that does not use `XDRSyncScript`.
   - The XDR integration instance incoming mapper is set to `Cortex XDR - Incoming Mapper` and the outgoing mapper is set to `Cortex XDR - Outgoing Mapper`.
+  - Mirroring impacts only incidents fetched after successfully enabling mirroring. If incidents were fetched without mirroring or with mirroring but with the incorrect mapper, enabling mirroring with the correct mapper now won't resolve issues for those incidents. The solution is to create a new instance from scratch.
```

Comment:
```suggestion
  - Mirroring impacts only incidents that were fetched after the mirroring was enabled for this instance. If incidents were fetched with the incorrect mapper, changing the mapper will not affect them. This can be solved by resetting the last fetch run and re-fetching the incidents.
```
This should be checked, but I think that just resetting last run can be a solution instead of creating a new instance.

Reply:
Done.